### PR TITLE
Talos body

### LINKED ISSUE #1755: Talos - Improve PR Body
Blocked on https://github.com/bbc/psammead/issues/1756 which gives context to what is bumped where

**Is your feature request related to a problem? Please describe.**
The current PR body, seen here https://github.com/bbc/psammead/pull/1734 isnt great

**Describe the solution you'd like**
Update the body of the PRs that are generated to have more information. If possible include a link to the PR that caused talos to run. If possible, add the changelog version diff to the new PR description telling us what this new PR is pulling in

image
which expands to 
image

**Additional context**

Testing:
The above description is straight forward which explains what is expected.

### PR DESCRIPTION
**Pull request by pharingee**
Resolves #1755

**Overall change:**
Improve Talos body

**Code changes:**
- Get changeling head based on published packages
- Update Talos body with Changelog heads of published packages

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by dr3 — COMMENTED** [review]
Just ran this with some dummy changes and it errored :O 
Made a branch `talos-body-testing` with [a few dummy changes](https://github.com/bbc/psammead/commit/619549c092c607a7bd47b2c9a7fa461fc22e8ca8)

https://ci.news.tools.bbc.co.uk/blue/organizations/jenkins/psammead/detail/talos-body-testing/2/pipeline/

```
* Creating Pull Request with title "Talos - Bump @bbc/gel-foundations, @bbc/psammead-styles, @bbc/psammead-brand"

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
```

File: scripts/talos/getChangelogHead/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  const changelogLines = readFile(path.join(packagePath, 'CHANGELOG.md')).split(
+    '\n',
+  );
+
+  const tableHeadRegex = /\|\s+?Version\s+?|\s+?Description\s+?\|/;
+  const tableHeadIndex = changelogLines.findIndex(line =>
+    line.match(tableHeadRegex),
+  );
+  const firstVersionIndex = changelogLines.findIndex(line =>
+    line.includes(packageVersions[0]),
```
Comment (dr3) [inline scripts/talos/getChangelogHead/index.js:33]: Not tested it, but could this get confused with `1.2.1` and `1.2.10`? So we miss changes `1.2.2` => `1.2.9`? May need to ensure its followed and lead by a non-number character? (likely a space or `|`, accounting for broken formatting :P)

Comment (pharingee) [reply scripts/talos/getChangelogHead/index.js:33]: True, I think a space suffix should fix it.

**Comment by dr3** [comment]
image

Chatted IRL, looks a bit odd 🙈 https://github.com/bbc/psammead/pull/2102

**Comment by pharingee** [comment]
Screenshot 2019-09-11 at 3 01 46 PM

It works if a new line is added before the table

**Review by dr3 — APPROVED** [review]
LGTM, great work

**Review by dr3 — COMMENTED** [review]
Whoops, @pharingee its failing jenkins 🙈

**Review by FK78 — APPROVED** [review]

**Comment by amywalkerdev** [comment]
```
Branch must update CHANGELOG.md in psammead

Branch must update package-lock.json in psammead

Branch must update package.json in psammead
```

ChangeScanner is flagging issues on build as the above have not been updated for this PR.

**Pull request merged**